app/Export/ExportServiceProvider.php

```diff
@@ -95,6 +99,20 @@ function (CommandBusInterface $commandBus, Application $app) {
                 return $commandBus;
             }
         );
+
+        $app['event_export_log_handler'] = $app->share(
+            function () {
+                return new StreamHandler(__DIR__ . '/../log/export.log');
```

Comment:
Are you sure this path is correct? Wouldn't this resolve to `/app/log/export.log` instead of `/log/export.log`?
Same for the other logger.